.circleci/config.yml

```diff
@@ -156,6 +156,38 @@ jobs:
       - store_artifacts:
           path: *TEST_RESULTS_DIR
 
+  # split off a job for the SDK package since it is separate
+  go-test-sdk:
+    docker:
+      - image: *GOLANG_IMAGE
+    environment:
+      <<: *ENVIRONMENT
+      GOTAGS: "" # No tags for OSS but there are for enterprise
+    steps:
+      - checkout
+      - restore_cache: # restore cache from dev-build job
+          keys:
+            - consul-modcache-v1-{{ checksum "go.mod" }}
+      - attach_workspace:
+          at: /go/bin
+      - run: mkdir -p $TEST_RESULTS_DIR
+      - run:
+          working_directory: sdk
+          command: |
+            PACKAGE_NAMES=$(go list ./... | circleci tests split --split-by=timings --timings-type=classname)
+            gotestsum --format=short-verbose --junitfile $TEST_RESULTS_DIR/gotestsum-report.xml -- -tags=$GOTAGS -cover -coverprofile=cov_sdk.part $PACKAGE_NAMES
+
+      # save coverage report parts
+      - persist_to_workspace:
```

Comment:
Nice to see these coverage reports all merge together and _just work_. I wasn't sure if it would.

Reply:
I didn't check that it actually did something sane with the output.

Reply:
It did! If you grab the artifact from here: https://circleci.com/gh/hashicorp/consul/129822#artifacts/containers/0 you can browse to the api and sdk packages to check coverage.